packages/blaze/builtins.js

```diff
@@ -226,6 +226,11 @@ Blaze.Each = function (argFunc, contentFunc, elseFunc) {
         });
       },
       changedAt: function (id, newItem, oldItem, index) {
```

Comment:
Yes, we have two additional diff levels right (see my original PR comment)
1. Data level
2. Field level in htmljs

I think 1 will being gone once we improve the original diff-ing which calls the observe handlers